ratis-netty/src/main/java/org/apache/ratis/netty/server/NettyServerStreamRpc.java

```diff
@@ -121,29 +146,30 @@ private void sendReply(DataStreamRequestByteBuf request, ChannelHandlerContext c
   private ChannelInboundHandler getServerHandler(){
     return new ChannelInboundHandlerAdapter(){
       @Override
-      public void channelRead(ChannelHandlerContext ctx, Object msg) {
+      public void channelRead(ChannelHandlerContext ctx, Object msg) throws IOException {
```

Comment:
Is there a need to add an exception handler in to handle IOException

```
 private ChannelInitializer<SocketChannel> getInitializer(){
    return new ChannelInitializer<SocketChannel>(){
      @Override
      public void initChannel(SocketChannel ch) {
        ChannelPipeline p = ch.pipeline();
        p.addLast(new DataStreamRequestDecoder());
        p.addLast(new DataStreamReplyEncoder());
        p.addLast(getServerHandler());
      }
    };
```?

Reply:
Let's make streaming working first and ignore exception handling for the moment.  We will work on exception handling in separated issues.  (Just like that we separated the retrying in this issue and didn't want RATIS-1082 becoming to big.)